doc/format-run.md

```diff
@@ -253,11 +268,10 @@ this, the following constraints on the sequence of stored keys follow:
 * The sequence must be non-empty.
 * The elements of the sequence must be non-decreasing.
 
-Like a compact-index file, an ordinary-index file starts with a 32 bit number
-that states the format version and, as a side effect, also determines the
-endianness used in the file.
+The type of compact indexes is denoted by the number 1. The type–version
+indicater in a serialised ordinary index is not padded.
```

Comment:
```suggestion
indicator in a serialised ordinary index is not padded.
```

Reply:
Fixed in 7374994e5e945729c5dfb45d79709e9f51674bcb.